src/Tpay/PayGroup.php

```diff
@@ -0,0 +1,16 @@
+<?php
+
+declare(strict_types=1);
+
+namespace CommerceWeavers\SyliusTpayPlugin\Tpay;
+
+class PayGroup
```

Comment:
```suggestion
final class PayGroup
```

WDYT? I can't imagine a situation where extending these classes is needed.